cms_delete_content/README.md

```diff
@@ -0,0 +1,113 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
But this file should be RST if we want that Odoo will be able to handle it, so better to rename to README.rst.